package.json

```diff
@@ -1,9 +1,13 @@
 {
   "scripts": {
     "heroku-prebuild": "cd client && yarn install && CI=true yarn build && cd .. && cd server && yarn install",
-    "start": "cd server && yarn start"
+    "client-start": "cd client && yarn start",
+    "server-start": "cd server && yarn start"
```

Comment:
The docker PR adds similar functionality but this is significantly simpler solution. Nice!